### PR TITLE
some change in js file

### PR DESCRIPTION
**Pull request by DmitryTravnikov**

### CODE REVIEW

File: test-rep.js [hunk 1]
```diff
@@ -1 +1,2 @@
 let str = 'this is a string';
+let str = 'the second strinh';
```
Comment (dtravnikov) [inline test-rep.js:2]: you have a mistake

Comment (DmitryTravnikov) [reply test-rep.js:2]: where?

Comment (dtravnikov) [reply test-rep.js:2]: on the second line

Comment (DmitryTravnikov) [reply test-rep.js:2]: ok, i'll solve it

**Pull request closed without merging**